site/src/Util/NotificationHydrator.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+namespace App\Util;
+
+/**
+ * This is the data class for creating a notification
+ * which will be then sent out to the RabbitMQ queue
+ *
+ * @copyright Aquib Baig <aquibbaig97@gmail.com>
+ *
+ * Class NotificationHydrator
+ */
+class NotificationHydrator
```

Comment:
Notification*Hydrator* is a strange name. This is just a plain old data class (POD), so how about `Notification`?